server/command.go

```diff
@@ -131,22 +132,26 @@ func (p *Plugin) runConnectCommand(user *model.User, extra *model.CommandArgs) (
 	}
 
 	oauthMsg := fmt.Sprintf(
-		zoomOAuthMessage,
-		*p.API.GetConfig().ServiceSettings.SiteURL, extra.ChannelId, trueString)
+		zoom.OAuthPrompt,
+		*p.API.GetConfig().ServiceSettings.SiteURL)
```

Comment:
Can we use a `GetSiteURL` method here? Looks like we use `p.siteURL` elsewhere too. Maybe just use that